### PR TITLE
Accessor CLI flags

### PR DESCRIPTION
**Pull request by vishalnayak**

### CODE REVIEW

File: api/auth_token.go [hunk 1]
```diff
@@ -124,9 +136,10 @@ func (c *TokenAuth) RevokePrefix(token string) error {
 	return nil
 }
 
-// RevokeSelf revokes the token making the call
-func (c *TokenAuth) RevokeSelf() error {
```
Comment (jefferai) [inline api/auth_token.go]: What happened to `RevokeSelf()`?  :sob:

Comment (vishalnayak) [inline api/auth_token.go]: It is not used.

Comment (jefferai) [inline api/auth_token.go]: Sure it is. Just not by us. We're not the only API consumers!

Comment (vishalnayak) [inline api/auth_token.go]: I will add it back :)

Comment (vishalnayak) [inline api/auth_token.go]: Done!

**Comment by jefferai** [comment]
Nice. :ship:

**Pull request merged**